packages/fluxible-router/tests/unit/lib/NavLink-test.js

```diff
@@ -111,16 +133,6 @@ describe('NavLink', function () {
             expect(ReactDOM.findDOMNode(link).getAttribute('href')).to.equal('/internal');
         });
 
-        it('should throw if href and routeName undefined', function () {
-            var navParams = {a: 1, b: 2};
-            expect(function () {
-                ReactTestUtils.renderIntoDocument(
-                    <MockAppComponent context={mockContext}>
-                        <NavLink navParams={navParams} />
-                    </MockAppComponent>
-                );
-            }).to['throw']();
-        });
```

Comment:
moved out into 2 other test suites (one for dev env, the other for prod env) to test diff behaviors in diff environments.